### DISCUSSION TITLE
커밋 올릴때 코멘트에 깃헙 포스트 번호를 같이 적으면

### DISCUSSION THREAD

**Discussion by RimWorldKorea** [RimWorldKorea/RMK#507] [일반]
해당 이슈 또는 디스커션에 자동으로 해당 커밋이 참조되어 보이기 때문에 추적하기가 쉬움

커밋 이름: 무슨무슨 수정
커밋 내용: #460

이런 식으로

참고로 내가 작업에 복귀하려면 2주 정도 더 걸릴듯?
그 때까진 메인 작업자로 활동 좀 부탁할께
이슈랑 저장소 수정하는건 계속 모니터링 중

@zilrty

**Comment by zilrty** [comment]
히익 빨리 돌아와줘잉
